Quickstarts/pom.xml

```diff
@@ -0,0 +1,55 @@
+<!-- Copyright (c) Microsoft. All rights reserved. --><!-- Licensed under the MIT license. See LICENSE file in the project root for full license information. -->
+<project>
+  <modelVersion>4.0.0</modelVersion>
+    <parent>
```

Comment:
I don't love this sample being defined in the root/Quickstarts path. Ideally, every sample in our repo would exist within the scope of the package that is being demonstrated. Since this is demonstrating a package we don't own, though, it is a special case. I'd consider moving this in with the IoT hub service client samples. This feature fits in the most with those samples